### PR TITLE
docs: Sandbox play with Mimir 

### PR DESCRIPTION
**Pull request by Jayclifford345**
#### What this PR does

This PR adds the Interactive sandbox metadata to the Play with Mimir tutorial. This tutorial can be tested here: https://killercoda.com/grafana-dev-testing/course/mimir/play-with-mimir

Here are the docs for the annotations I added: https://github.com/grafana/killercoda/blob/staging/docs/transformer.md

This is part of a larger effort to make our docs more interactive. Happy to jump on a call to discuss further as needed.

#### Checklist

- [ ] Tests updated.
- [x] Documentation added.
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`.
- [ ] [`about-versioning.md`](https://github.com/grafana/mimir/blob/main/docs/sources/mimir/configure/about-versioning.md) updated with experimental features.

### CODE REVIEW

**Comment by Jayclifford345** [comment]
@jdbaldry hello my good sir! Does Mimir have a different docs checker than the other repos? Just wanted to check whether it should still be the case to include images directly within the repo

**Comment by jdbaldry** [comment]
> @jdbaldry hello my good sir! Does Mimir have a different docs checker than the other repos? Just wanted to check whether it should still be the case to include images directly within the repo

Good spot, yeah, they have different rules in Mimir.

I should really work with @tacole02 to migrate them to using the media storage.

**Comment by Jayclifford345** [comment]
aaah dang good to know. I will re-add the image back in and remove my banner to save repo space

**Comment by jdbaldry** [comment]
Think the other linting error is just formatting with `prettier` (or running the command from the CI output)

https://github.com/grafana/mimir/actions/runs/10791026737/job/29927546365?pr=9247

**Comment by tacole02** [comment]
Let's do it, @jdbaldry 😄

File: docs/sources/mimir/get-started/play-with-grafana-mimir/index.md
Comment (tacole02) [inline docs/sources/mimir/get-started/play-with-grafana-mimir/index.md:65]:
```suggestion
It's a fully configured environment with all dependencies already installed.
```

File: docs/sources/mimir/get-started/play-with-grafana-mimir/index.md
Comment (tacole02) [inline docs/sources/mimir/get-started/play-with-grafana-mimir/index.md:203]:
```suggestion
   1. Type `count(up == 0)` in the **Metrics browser** query field. This currently shows `no data` since all instances are running.
```

File: docs/sources/mimir/get-started/play-with-grafana-mimir/index.md
Comment (tacole02) [inline docs/sources/mimir/get-started/play-with-grafana-mimir/index.md:206]:
```suggestion
   1. Select `New evaluation group` and type `example-group` in the **Group name** field. Set the evaluation interval to `30s`.
```

**Review by tacole02 — APPROVED** [review]
LGTM 😄

**Comment by Jayclifford345** [comment]
Amazing! Thanks @tacole02 ready to merge when you are. I checked the lint test that was failing and looks to be down to:
us using `` rather than "". This is expected on our end for the regex parser.

**Comment by tacole02** [comment]
🚢 🚢 ✨

**Comment by Jayclifford345** [comment]
I don't have permission to merge sadly!

**Review by seizethedave — APPROVED** [review]

**Comment by seizethedave** [comment]
CI linter complains about some quotes:
https://github.com/grafana/mimir/actions/runs/10991534562/job/30514113632

**Comment by seizethedave** [comment]
I don't think the PR can be merged until that's rectified. Hmm...

**Comment by tacole02** [comment]
Hmm, it was my understanding that these quotes were necessary. @Jayclifford345 please confirm!

**Comment by Jayclifford345** [comment]
Hi @seizethedave,
This is part of the killer coda metadata for our regex parser. @jdbaldry I belive we looked at this together and if we swapped to string literal the parser failed which is why we have to use double "" is that correct?

File: docs/sources/mimir/get-started/play-with-grafana-mimir/index.md
Comment (jdbaldry) [inline docs/sources/mimir/get-started/play-with-grafana-mimir/index.md:35]: I think this should be fine as the linter wants, I think the issue with parsing must have been something else.

I've also escaped the period in the regular expression because we want to match it literally rather than match any character. I believe with YAML double quoted strings we need to escape the backslash

```suggestion
      - regexp: "tutorial-architecture\\.png"
```

**Comment by Jayclifford345** [comment]
Hi all, can confirm the change did not break the transformer so hopefully all good to go. With tests passed

**Comment by Jayclifford345** [comment]
Hey, all just wondering if we can now merge? @seizethedave and @tacole02? Many thanks in advance!

**Comment by tacole02** [comment]
@seizethedave would you mind merging this if it looks okay to you? I don't have merge rights for mimir.

**Comment by Jayclifford345** [comment]
thanks all!

**Pull request merged**
